### PR TITLE
[MM-29562]: add e2e tests to archiveCommandCmdF

### PR DESCRIPTION
**Pull request by simross**
**Summary**

Adds e2e tests for archiveCommandCmdF.

**Ticket Link**

Fixes mattermost/mattermost-server#15940

JIRA: https://mattermost.atlassian.net/browse/MM-29562

### CODE REVIEW

File: commands/command_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		Name:        api4.GenerateTestTeamName(),
+		Email:       s.th.BasicUser.Email,
+		Type:        model.TEAM_OPEN,
+	})
+	s.Require().Nil(appErr)
+
+	_, appErr = s.th.App.AddUserToTeam(teamOfBasicUser.Id, s.th.BasicUser.Id, "")
+	s.Require().Nil(appErr)
+
+	s.RunForSystemAdminAndLocal("Archive nonexistent command", func(c client.Client) {
```
Comment (streamer45) [inline commands/command_e2e_test.go:31]: We can probably use `RunForAllClients` for this one too.

**Review by streamer45 — APPROVED** [review]
Thank you for your contribution @simross! PR looks very good :+1: 
I've left a minor suggestion.

File: commands/command_e2e_test.go (hunk 1 above)
Comment (simross) [reply commands/command_e2e_test.go]: fixed

**Review by lindy65 — APPROVED** [review]
Thanks @simross - looks good to me 👍

**Review by mgdelacroix — APPROVED** [review]
LGTM! Thanks @simross!! :tada:

**Pull request merged**